### PR TITLE
RCP Documentation

### PR DESCRIPTION
**Pull request by emizan76**

### CODE REVIEW

File: training_rules.adoc [hunk 1]
```diff
@@ -475,13 +475,51 @@ Each benchmark result is computed by dropping the fastest and slowest runs, then
...
+
+* Reference implementations provide at least 2N epoch convergence numbers, where N is the number of submission runs needed for each benchmark. Since convergence is affected by batch size (larger batch size means slower convergence), reference implementations provide convergence data for a few different batch sizes.
+* Using the mean and standard deviation of the reference convergence we apply a 1-sided student t-test with p-value=0.05 to find the maximum acceptable speedup for submission convergence.
```
Comment (matthew-frank) [inline training_rules.adoc]: "student t-test" => "[independent two-sample Student's t-test with unequal sample sizes, similar variances](https://en.wikipedia.org/wiki/Student%27s_t-test#Equal_or_unequal_sample_sizes,_similar_variances_(1/2_%3C_sX1/sX2_%3C_2)"

File: training_rules.adoc [hunk 2]
```diff
@@ -475,13 +475,51 @@ Each benchmark result is computed by dropping the fastest and slowest runs, then
...
+** If the submission batch size is smaller that the batch size of any RCP AND the convergence test against the RCP with the minimum batch size fails, then again the submitter must provide the missing RCPs by running the reference implementation with their batch size.
+
+Please refer to the related Appendix for examples that shed light to the RCP process.
+
+
+=== RCP Rules and Guidelines
+
+Submitters are encouraged to run the RCP checker script prior to their submission to make sure they do not violate RCP limits.
+
+If a submission fails the RCP test, such as S2 in the Appendix, they have the option to submit with the --rcp_bypass parameter. This will allow the submission to go through but at review time the submitter should be able to justify why their submission is valid while it failed the RCP test.
```
Comment (matthew-frank) [inline training_rules.adoc]: "go through but at review time the submitter should be able to justify why their submission is valid while it failed the RCP test."
=>
"upload, but the submitter must notify the results chair, and prepare for the audit process described in the [MLPerf Submission Rules document](https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#auditing)."

(Section 6.3 of the Submission Rules is currently just marked "TBD", but the proposed text is https://github.com/mlcommons/policies/pull/79/files)

File: training_rules.adoc [hunk 3]
```diff
@@ -475,13 +475,51 @@ Each benchmark result is computed by dropping the fastest and slowest runs, then
...
+
+If a submission is missing the RCP for the batch size they are submitting, such as S4 and S6 in the Appendix they must provide the missing convergence points by making a PR in the logger. If not, the submission will not be accepted. All missing RCPs are due at submission time.
+
+During hyperparameter borrowing, borrowers can use hyperparameters from submissions that passed or failed the RCP test. If their submission fails to pass the RCP test they can have it go through by using --rcp-bypass and then be ready to defend it during the review process.
```
Comment (matthew-frank) [inline training_rules.adoc]: "go through by using --rcp-bypass and then be ready to defend it during the review process"
=>
"upload by using --rcp-bypass, and then prepare for the audit described in the [MLPerf Submission Rules document](https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#auditing)."

File: training_rules.adoc (hunk 1 above)
Comment (emizan76) [reply training_rules.adoc]: Fixed

File: training_rules.adoc (hunk 3 above)
Comment (emizan76) [reply training_rules.adoc]: Fixed

File: training_rules.adoc (hunk 2 above)
Comment (emizan76) [reply training_rules.adoc]: Fixed

**Review by matthew-frank — APPROVED** [review]
Looks great to me.  Thank you!

File: training_rules.adoc [hunk 4]
```diff
@@ -475,13 +475,51 @@ Each benchmark result is computed by dropping the fastest and slowest runs, then
...
+|===
+| Benchmark | Epoch reporting 
+
+| RN50 | Epoch 
+| BERT | Eval sample (integer) 
+| DLRM | Epoch / 20.0 
+| SSD | Epoch 
+| Mask-RCNN | Epoch 
+| RNN-T | Epoch 
+| UNET3D | Eval sample
```
Comment (mmarcinkiewicz) [inline training_rules.adoc]: UNET3D is using Epochs as well. There was a discussion to move to samples but finally the decision was made to stick with epochs. More details here https://github.com/mlcommons/training/pull/458 and here https://github.com/mlcommons/training_policies/blob/master/training_rules.adoc#94-quality-measure

Comment (emizan76) [reply training_rules.adoc]: Oh Ok, I will fix it then

**Review by johntran-nv — APPROVED** [review]

**Pull request merged**